package.json

```diff
@@ -47,6 +47,7 @@
     "xo": "*"
   },
   "xo": {
-    "esnext": true
+    "esnext": true,
+    "space": true
```

Comment:
Sorry, I was confused because you had spaces instead of tabs in the `meow` helper and it confused my editor.